tpm2_pytss/crypto.py

```diff
@@ -88,6 +88,11 @@ def key_from_encoding(data):
         key = cert.public_key()
     elif sdata.startswith(b"-----BEGIN PUBLIC KEY-----"):
         key = load_pem_public_key(data, backend=default_backend())
+    elif sdata.startswith(b"-----BEGIN RSA PRIVATE KEY-----") or sdata.startswith(
+        b"-----BEGIN EC PRIVATE KEY-----"
+    ):
+        pkey = load_pem_private_key(data, password=None, backend=default_backend())
```

Comment:
We probably want to plumb password through

Reply:
sure, I'll add a password argument to from_pem for both the public and private versions

Reply:
Done